README.md

```diff
@@ -467,26 +467,36 @@ pyATS Docker images created using this package features the following directory
 structure:
 
 ```text
-/venv
+/pyats
     Directory where the Python virtual environment is created. All Python
     packages (including pyATS) specified in the build YAML file are installed
-    into here.
+    into here. Additionally acts as the workspace, where all files and
+    repositories specified in the YAML build file gets copied to. Set as the
+    Docker working directory.
 
-/workspace
```

Comment:
same with above. no more `/workspace`? I assume we need to update docs and etc. when it comes to the impact where we need to update, I'm feeling not good idea to change this... or will you take care of those and already knew not impacted...?

Reply:
`/workspace` and `/venv` were merged to become `/pyats` many months ago, that's not the purpose of this PR. The readme has just been updated to reflect this now. I'll go double check that all the paths are correct in the readme. This PR only adds the `repos.json` file and modifies the `manifest.json` file, which are records of available git repos and manifest files, but do not impact the actual execution of the final docker image.